scss/_patterns_aside.scss

```diff
@@ -0,0 +1,58 @@
+@import 'settings';
+
+@mixin vf-p-aside {
+  .p-aside {
+    align-self: flex-start;
+    border-top: 1px solid $color-mid-light;
+    flex: 0 0 16rem;
+    font-size: .875rem;
+    padding: 0 $sp-medium;
+
+    @media (min-width: $breakpoint-medium) {
+      border-left: 1px solid $color-mid-light;
+      border-top: 0;
+      margin-top: $sp-xxx-large * 2;
```

Comment:
Why do we need such a large margin-top? This is probably going to have to be overridden in most other cases.

Reply:
This was part of the pattern in vanilla-docs-theme - I didn't amend it.

Reply:
Again I think this should also be removed. It limits the use of this pattern in other situations.